src/Modules/Manage/Dnn.PersonaBar.Users/Users.Web/src/_exportables/src/components/common/Password/index.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { Component,PropTypes  } from "react";
+import SingleLineInputWithError from "dnn-single-line-input-with-error";
+import Localization from "localization";
+import passwordStrength from "utils/PasswordStrength";
+import {CommonUsersService as UserService} from "../../../services";
+
+import "./style.less";
+
+class Password extends Component {
+    constructor(props) {
+        super(props);
+        this.passwordStrengthOptions = {
+            minLength:7,
+            minNumberOfSpecialChars:1,
+            validationExpression:""        
+        };
+    }
+
+    componentDidMount(){
+        
+        UserService.getPasswordStrengthOptions((data)=>{
+            this.passwordStrengthOptions = data;
+        });
+    }
+
+    _getStrength(){
+        let password = this.props.UserDetails.password;
+        let pStrength = passwordStrength(password,this.passwordStrengthOptions);
+        if (password.length <= 2 ) {
+            return null;
+        }
+        if (pStrength.rating <3 ) {
+            return "weak";
+        }
+        if (pStrength.rating < 5) {
+            return "fair";
+        }
+        if (pStrength.rating >= 5) {
+            return "strong";
+        }
+        return "weak";
```

Comment:
I'd introduce some object with constants for the `weak / fair / strong` values. What do you think?